RecoEgamma/ElectronIdentification/interface/ElectronMVAEstimatorRun2.h

```diff
@@ -47,13 +71,13 @@ class ElectronMVAEstimatorRun2 : public AnyMVAEstimatorRun2Base {
   std::vector<int> nVariables_;
 
   // Data members
-  std::vector<std::unique_ptr<const GBRForest>> gbrForests_;
+  std::vector<const GBRForest*> gbrForests_;
```

Comment:
Why is the smart pointer removed? (was not clear to me in the commit message)

Reply:
Because you can't use a `unique_ptr<T>` if T is only forward declared. But that was an over-optimization anyway, I included now `GBRForest.h` again, and generating the dictionaries for the `ElectronMVAEstimatorRun2` is still very fast.